### DISCUSSION TITLE
Is it possible to use OData internally?

### DISCUSSION THREAD

**Discussion by Diaz15h** [OData/AspNetCoreOData#642] [Q&A]
Here I would like to use Odata without going through http
Something like that `mode.ApplyTo(dbContext.people.AsQueryable(), query)`
Notes: There is a lack of documentation on OData v8 with Net core 6.
Thanks

**Comment by julealgon** [comment]
This should be moved to discussions.

**Comment by Diaz15h** [comment]
Yes indeed it should be in the discussion section
